modules/tinymce/src/core/test/ts/browser/keyboard/SpaceKeyTest.ts

```diff
@@ -1,5 +1,5 @@
 import { Keys } from '@ephox/agar';
-import { beforeEach, context, describe, it } from '@ephox/bedrock-client';
+import { assert, beforeEach, context, describe, it } from '@ephox/bedrock-client';
```

Comment:
This should be using `chai` instead of the legacy/deprecated Bedrock assert function (we should delete this next bedrock major). As a side note, we'll also need to use `deepEqual` below.

- [ ] Use chai assertions